### PR TITLE
Get rid of Simple Server Reference

### LINKED ISSUE #442: Get Rid of Simple Server Section of Hosting Doc
### What You Are Seeing?
Get rid of simple server section of [host packages internally](https://docs.chocolatey.org/en-us/features/host-packages#chocolatey.server-setup) page.  

### What is Expected?
We do not advertise simple server as a setup option any longer.

### PR DESCRIPTION
**Pull request by ryanrichter94**
## Description Of Changes
Get rid of simple server reference as well as general markdown linting.

## Motivation and Context
We do not support simple server and general formatting upkeep.

## Testing

* [X] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [X] Minor documentation fix (typos etc.).
* [X] Major documentation change (refactoring, reformatting or adding/removing documentation to existing page).
* [ ] New documentation page added.

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue

Fixes #442

### CODE REVIEW

**Comment by pauby** [comment]
@ryan there are a few mentions of 'simple server' in the docs. I think this is confusing. Simple server refers to one thing (it's a server not serverS) but in the context of the docs it's actually a basic nuget server hosting. Simple Server will also mean something to the audience. If we got away from that wording completely I think it would help us all ... move on 😏 

What do you think about changing it to read like that?

File: input/en-us/features/host-packages.md [hunk 1]
```diff
@@ -97,8 +97,8 @@ Permissions can be interesting with a file share. If you are using machine accou
 
 A great read on your options can be found at the following Stack Exchange links:
 
-* https://serverfault.com/q/135867/79259
-* https://serverfault.com/q/41130/79259
+* <https://serverfault.com/q/135867/79259>
+* <https://serverfault.com/q/41130/79259>
```
Comment (pauby) [inline input/en-us/features/host-packages.md:101]: I'm not sure why this change has been made? Do you have an extension that adds this around links?

Comment (ryanrichter94) [reply input/en-us/features/host-packages.md:101]: @pauby This change is from [markdownlint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint). Specifically around resolving [issue MD034 - Bare URL used](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md034---bare-url-used).

Comment (pauby) [reply input/en-us/features/host-packages.md:101]: 👍 It isn't needed here as Statiq converts the links without any issue.

**Comment by gep13** [comment]
@ryanrichter94 what is the status of this PR? What was the reason that this was switched to draft?

**Comment by gep13** [comment]
@ryanrichter94 @pauby gentle ping on this. Thanks

**Comment by gep13** [comment]
@ryanrichter94 @pauby Any updates on this? Would be good to get this updated and merged.

**Comment by pauby** [comment]
Not from me but I'll keep this around so that I know to come back to it.
